wiki.php

```diff
@@ -18,19 +19,28 @@
 define('WEBHOOKS_VOCABULARY_WEBHOOK', "http://yeswiki.net/_vocabulary/webhook");
 define('WEBHOOKS_VOCABULARY_TEST', "http://yeswiki.net/_vocabulary/webhook-test");
 
+define('WEBHOOKS_ACTIVITYPUB_PUBLIC_URI', "https://www.w3.org/ns/activitystreams#Public");
+
 //////////////////
 // Configs
 //////////////////
 
 // Available JSON formatting to ease integration with other services
 // If you add new formats, you will also need to modify format_json_data() and the lang file(s)
-$wakkaConfig['WEBHOOKS_FORMATS'] = getConfigValue('WEBHOOKS_FORMATS', [
+$wakkaConfig['webhooks_formats'] = getConfigValue('webhooks_formats', [
     WEBHOOKS_FORMAT_RAW => 'WEBHOOKS_FORMAT_RAW',
+    WEBHOOKS_FORMAT_ACTIVITYPUB => 'WEBHOOKS_FORMAT_ACTIVITYPUB',
     WEBHOOKS_FORMAT_MATTERMOST => 'WEBHOOKS_FORMAT_MATTERMOST',
     WEBHOOKS_FORMAT_SLACK => 'WEBHOOKS_FORMAT_SLACK',
 ], $wakkaConfig);
 
+// If no actor field is defined semantically, this value will be used
+$wakkaConfig['webhooks_activitypub_default_actor'] = getConfigValue('webhooks_activitypub_default_actor', '', $wakkaConfig);
+
+// If an actor is defined, we can override the base URL with this config
+$wakkaConfig['webhooks_activitypub_actors_base_url'] = getConfigValue('webhooks_activitypub_actors_base_url', '', $wakkaConfig);
+
 // Bot config (works for Mattermost if username and profile picture override is enabled)
 // See https://docs.mattermost.com/developer/webhooks-incoming.html
-$wakkaConfig['WEBHOOKS_BOT_NAME'] = getConfigValue('WEBHOOKS_BOT_NAME', "YesWiki Bot", $wakkaConfig);
-$wakkaConfig['WEBHOOKS_BOT_ICON'] = getConfigValue('WEBHOOKS_BOT_ICON', "https://yeswiki.net/files/PageHeader_yeswikiprovisoire_vignette_97_97_20181206153605_20181206154004.png", $wakkaConfig);
\ No newline at end of file
+$wakkaConfig['webhooks_bot_name'] = getConfigValue('webhooks_bot_name', "YesWiki Bot", $wakkaConfig);
+$wakkaConfig['webhooks_bot_icon'] = getConfigValue('webhooks_bot_icon', "https://yeswiki.net/files/PageHeader_yeswikiprovisoire_vignette_97_97_20181206153605_20181206154004.png", $wakkaConfig);
```

Comment:
It's a detail, but maybe use local, relative url like `tools/webhooks/presentation/images/default-bot.png` with a image like https://pixabay.com/illustrations/bot-icon-robot-automated-cyborg-2883144/

Reply:
Done :)